apps/studio/components/interfaces/Auth/RateLimits/RateLimits.tsx

```diff
@@ -408,8 +408,8 @@ const RateLimits = () => {
                   <FormSectionLabel
                     description={
                       <p className="text-foreground-light text-sm">
-                        Number of sign up and sign-in requests that can be made per hour per IP
-                        address (excludes anonymous users).
+                        Number of sign up and sign-in requests that can be made per IP address
+                        (excludes anonymous users).
```

Comment:
We don't say anywhere that this is in a 5 minute interval though right?
```suggestion
                        Number of sign up and sign-in requests that can be made in a 5 minute interval per IP address
                        (excludes anonymous users).
```

Reply:
But the hint below the editable field shows the per hour rate. I am not sure where you are getting this 5 minutes from. 🤔

Reply:
just chatted with Joel - our docs need some updating, alaister is right here RE the 5 minute interval although we currently don't mention about it in our docs

Reply:
Updated here: https://github.com/supabase/supabase/pull/29176/